module/workflow/src/Infrastructure/Persistence/Projector/DbalStatusCreatedEventProjector.php

```diff
@@ -40,6 +48,7 @@ public function __invoke(StatusCreatedEvent $event): void
                 'name' => $this->serializer->serialize($event->getName()->getTranslations()),
                 'description' => $this->serializer->serialize($event->getDescription()->getTranslations()),
                 'color' => $event->getColor()->getValue(),
+                'index' => $maxIndex + 1,
```

Comment:
this can be done in one query. eg :

```
$qb = $this->connection->createQueryBuilder();
        $qb->insert(self::TABLE)
            ->set('index', '(SELECT max(x)+1 FROM y)')
...
```